src/main/java/net/syscon/elite/service/OffenderAddressService.java

```diff
@@ -29,9 +29,10 @@
     @VerifyOffenderAccess
     public List<AddressDto> getAddressesByOffenderNo(@NotNull String offenderNo) {
         final var offenderBookings = offenderBookingRepository.findByOffenderNomsIdAndActiveFlag(offenderNo, "Y");
-        if(offenderBookings.size() > 1) throw new IllegalStateException(String.format("More than one active booking was returned for offender number %s\n", offenderNo));
+        if (offenderBookings.size() > 1)
+            throw new IllegalStateException(String.format("More than one active booking was returned for offender number %s\n", offenderNo));
 
-        final var offenderBooking = offenderBookings.stream().findFirst().orElseThrow(EntityNotFoundException.withMessage(String.format("No active offender bookings found for offender number %s\n",offenderNo)));
+        final var offenderBooking = offenderBookings.stream().findFirst().orElseThrow(EntityNotFoundException.withMessage(String.format("No active offender bookings found for offender number %s\n", offenderNo)));
         final var offenderRootId = offenderBooking.getOffender().getRootOffenderId();
```

Comment:
could the offender of the booking be null? probably not, but is it worth checking?

Reply:
No it can't. The mapping requires it.

Reply:
`    @ManyToOne(optional = false)
    @JoinColumn(name = "OFFENDER_ID", nullable = false)
    private Offender offender;
`